### ISSUE TITLE
google_analytics_3 raises The requested URL /analytics/v3/data/ga/ was not found on this server

### ISSUE THREAD

**Issue by hidekoji** [MarkEdmondson1234/googleAnalyticsR#286]
## What goes wrong

When I call `googleAnalyticsR::google_analytics_3`, it throws <p>The requested URL <code>/analytics/v3/data/ga/</code> was not found on this server.  

## Steps to reproduce the problem

Run below code in RStudio

## Expected output

Query result.

## Actual output

Below is the error I got when I tried it with RStudio.

```r
> options(googleAuthR.verbose=2)
> googleAnalyticsR::google_analytics_3(XXXXX, start="2019-10-01", end="2019-10-10",  dimensions = 'ga:userType', metrics = 'ga:newUsers', max_results = 100)
2019-10-29 17:52:55> Request: https://www.googleapis.com/analytics/v3/data/ga/?ids=ga:XXXXX&start-date=2019-10-01&end-date=2019-10-10&metrics=ga:newUsers&dimensions=ga:userType&samplingLevel=DEFAULT&start-index=1&max-results=100&output=json
2019-10-29 17:52:55> Request Status Code: 404
Error : lexical error: invalid char in json text.
                                       <!DOCTYPE html> <html lang=en> 
                     (right here) ------^

<!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL <code>/analytics/v3/data/ga/</code> was not found on this server.  <ins>That’s all we know.</ins>
2019-10-29 17:52:55> API returned error: API error: returned web page that has been opened in your default browser if possible
2019-10-29 17:52:55> No retry attempted: API error: returned web page that has been opened in your default browser if possible
Error: lexical error: invalid char in json text.
                                       <!DOCTYPE html> <html lang=en> 
                     (right here) ------^
Called from: parse_string(txt, bigint_as_char)```

## Session Info

```r
sessionInfo()
R version 3.6.0 (2019-04-26)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.2                lubridate_1.7.4           countrycode_1.1.0         lattice_0.20-38          
 [5] tidyr_1.0.0               googleCloudStorageR_0.5.1 ps_1.3.0                  zoo_1.8-6                
 [9] assertthat_0.2.1          zeallot_0.1.0             digest_0.6.21             foreach_1.4.7            
[13] utf8_1.1.4                R6_2.4.0                  plyr_1.8.4                backports_1.1.5          
[17] reprex_0.3.0              evaluate_0.14             httr_1.4.1                ggplot2_3.2.1            
[21] pillar_1.4.2              rlang_0.4.0               lazyeval_0.2.2            curl_4.2                 
[25] rstudioapi_0.10           data.table_1.12.2         callr_3.3.2               whisker_0.4              
[29] googleAuthR_1.1.1       rmarkdown_1.16            urltools_1.7.3            stringr_1.4.0            
[33] triebeard_0.3.0           munsell_0.5.0             proxy_0.4-23              broom_0.5.2.1            
[37] xfun_0.10                 compiler_3.6.0            pkgconfig_2.0.3           askpass_1.1              
[41] clipr_0.7.0               htmltools_0.4.0           openssl_1.4.1             tidyselect_0.2.5         
[45] tibble_2.1.3              anonymizer_0.2.2          dtw_1.21-3                codetools_0.2-16         
[49] fansi_0.4.0               crayon_1.3.4              dplyr_0.8.3               grid_3.6.0               
[53] nlme_3.1-139              jsonlite_1.6              gtable_0.3.0              exploratory_5.4.0.9      
[57] lifecycle_0.1.0           magrittr_1.5              scales_1.0.0              zip_2.0.4                
[61] cli_1.1.0                 stringi_1.4.3             reshape2_1.4.3            fs_1.3.1                 
[65] vctrs_0.2.0               generics_0.0.2            prediction_0.3.14         iterators_1.0.12         
[69] tools_3.6.0               glue_1.3.1                purrr_0.3.2               RGoogleAnalytics_0.1.6   
[73] processx_3.4.1            parallel_3.6.0            yaml_2.2.0                colorspace_1.4-1         
[77] gargle_0.4.0              googleAnalyticsR_0.7.0    memoise_1.1.0             knitr_1.25    
```

**Comment by MarkEdmondson1234** [comment]
Temporary glitch with the API?  It seems to work now?

```r
googleAnalyticsR::google_analytics_3(ga_id, start="2019-10-01", end="2019-10-10",  dimensions = 'ga:userType', metrics = 'ga:newUsers', max_results = 100)
2019-10-30 12:38:34> Request to profileId: 81416156 (Live Blog)
2019-10-30 12:38:34> Fetched: userType newUsers. [2] total results out of a possible [2], Start-Index: 1
           userType newUsers
1       New Visitor     1948
2 Returning Visitor        0
```

**Comment by MarkEdmondson1234** [comment]
Thanks to @octaviancorlade who has put a pull in to remove the trailing slash, that may be intermittently causing this issue.

**Comment by MarkEdmondson1234** [comment]
Has this corrected the issue for everyone?  Try GitHub version

`remotes:install_github("MarkEdmondson1234/googleAnalyticsR")`

**Comment by octaviancorlade** [comment]
Thanks for merging this so quickly! It looks like this has corrected the issue for us so far, also when the error appears I am able to consistently reproduce the error using the version in CRAN and it gets fixed installing again from GitHub.

**Comment by MarkEdmondson1234** [comment]
Ok great.  Weird it started recently, but glad its a quick fix.

**Comment by MarkEdmondson1234** [comment]
This fix is on CRAN now, download 0.7.1
